mchf-eclipse/drivers/audio/rtty.c

```diff
@@ -717,6 +717,14 @@ int DigiModes_TxBufferPutChar(uint8_t c)
     return ret;
 }
 
+void DigiModes_TxBufferPutSign(const char* s)
```

Comment:
Why not process as strings here, your input in your datastructure is are strings anyway.